public/static/admin/config.yml

```diff
@@ -199,6 +199,8 @@ collections:
     label_singular: "Blog Post"
     folder: "content/blog/posts"
     create: true
+    slug: "{{year}}-{{month}}-{{day}}_{{title}}_{{authors}}"
+    preview_path: "blog/{{year}}/{{month}}/{{slug}}"
```

Comment:
Do we need this? I don't think the URLs for our blog posts will contain the date like this.